crates/runner-host/crates/web-server/src/lib.rs

```diff
@@ -120,10 +120,9 @@ async fn handle(mut ws: WebSocket, client: ClientInput) {
     });
     cmd_sender.send(Command::Connected).await.unwrap();
     let board_config = Command::BoardConfig {
-        components: vec![
-            Component::Button { id: BUTTON_ID },
-            Component::MonochromeLed { id: LED_ID },
-        ],
+        components: vec![Component::Button { id: BUTTON_ID }, Component::MonochromeLed {
+            id: LED_ID,
+        }],
```

Comment:
This looks worse. Seems to be https://github.com/rust-lang/rustfmt/issues/3370#issuecomment-1288226646.

Reply:
I agree. I didn't find a workaround. Let's keep it like that for now.